instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/core/CoreRerouteTest.kt

```diff
@@ -78,16 +79,22 @@ class CoreRerouteTest : BaseTest<EmptyTestActivity>(EmptyTestActivity::class.jav
         )
     }
 
+    override fun setupMockLocation(): Location {
+        val mockRoute = MockRoutesProvider.dc_very_short(activity)
+        return mockLocationUpdatesRule.generateLocationUpdate {
+            latitude = mockRoute.routeWaypoints.first().latitude()
+            longitude = mockRoute.routeWaypoints.first().longitude()
+        }
+    }
+
     @Test
     fun reroute_completes() {
         // prepare
         val mockRoute = MockRoutesProvider.dc_very_short(activity)
-
         val originLocation = mockLocationUpdatesRule.generateLocationUpdate {
             latitude = mockRoute.routeWaypoints.first().latitude()
             longitude = mockRoute.routeWaypoints.first().longitude()
         }
```

Comment:
Are we ok to re-push same location as in `setupMockLocation`? Probably yes (safer) but was curious because noticed that was removed from other tests below 👇

Reply:
Actually would like to be minimal unless there is a reason not to.

Updated!